packages/cards-editor-config-writer/src/index.ts

```diff
@@ -0,0 +1,85 @@
+import { join } from 'path';
+import type { Editor } from 'mem-fs-editor';
+import { create as createStorage } from 'mem-fs';
+import { create } from 'mem-fs-editor';
+import type { Package, ManifestNamespace } from '@sap-ux/project-access';
+import { UI5Config } from '@sap-ux/ui5-config';
+
+export type Manifest = ManifestNamespace.SAPJSONSchemaForWebApplicationManifestFile & { [key: string]: unknown };
+/**
+ * Updates the package.json file to include the cards editor middleware and the start-cards-generator script.
+ *
+ * @param basePath - The path to the project root
+ * @param fs - Mem-fs editor instance
+ */
+function updatePackageJson(basePath: string, fs: Editor) {
+    const packageJsonPath = join(basePath, 'package.json');
+    if (!fs.exists(packageJsonPath)) {
+        throw new Error('package.json not found');
+    }
+
+    const packageJson = (fs.readJSON(packageJsonPath) ?? {}) as Package;
+
+    packageJson.devDependencies ??= {};
+    packageJson.devDependencies['@sap-ux/cards-editor-middleware'] ??= '*';
+
+    packageJson.scripts ??= {};
+    packageJson.scripts['start-cards-generator'] = `fiori run --open \"test/flpGeneratorSandbox.html#Cards-generator\"`;
```

Comment:
Please check this one again because I think you might need `\\"` to actually get the `\"` into your string

Reply:
I saw that one when looking at the 5 sonar issues reported for this PR. Please address those 5 as well

Reply:
Thanks for pointing this out. We don't need \\ ,  `fiori run --open "test/flpGeneratorSandbox.html#Cards-generator"` is sufficient.